src/interfaces/IMetaMorpho.sol

```diff
@@ -95,8 +95,12 @@ interface IMetaMorphoBase {
     /// @notice Revokes the pending cap of the market defined by `id`.
     function revokePendingCap(Id id) external;
 
-    /// @notice Submits a forced market removal from the vault, potentially losing all funds supplied to the market.
+    /// @notice Submits a forced market removal from the vault, losing all funds supplied to the market.
```

Comment:
```suggestion
    /// @notice Submits a forced market removal from the vault, losing all funds supplied to the market unless the market is re-enabled later on.
```

Reply:
But I don't like this because it's pointing out funds can be recovered later on except it funds will be distributed pro-rata to the shares when recovered, not pro-rata to the shares when lost...